src/app/services/rate.service.ts

```diff
@@ -238,7 +238,7 @@ export class RateService {
 
       if (step.items != null) {
         const item = data.itemEntities[step.itemId];
-        if (item.rocketCapacity)
+        if (item.rocketCapacity?.nonzero())
```

Comment:
Should probably remove the zero values from the data as well, they should be left off the object if they would be zero, but this is the more important half of the fix.